distribution/build-configuration/version.info

```diff
@@ -1,5 +1,5 @@
-#Thu, 10 Jan 2019 15:15:21 -0500
+#Fri, 01 Mar 2019 03:16:49 +0530
 bindaas.framework.version.major=3
 bindaas.framework.version.minor=3
-bindaas.framework.version.revision=8
-bindaas.build.date=201901101515
+bindaas.framework.version.revision=9
+bindaas.build.date=201903010316
```

Comment:
Avoid committing this file. It auto-increments. But we commit this file only when we increment the version.

Reply:
Sure.